src/library_dylink.js

```diff
@@ -646,7 +651,11 @@ var LibraryDylink = {
     return loadModule();
   },
 
-#if STACK_OVERFLOW_CHECK >= 2
+#if STACK_OVERFLOW_CHECK >= 2 && USE_PTHREADS
+  // With USE_PTHREADS we load libraries before we are running a pthread and
+  // therefore before we have a stack.  Instead we calling `__set_stack_limits`
```

Comment:
I think `calling` is a typo, but I'm not sure what it should be - who calls what?